content/en/docs/migration/ot_overview.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Migrating from OpenTelemetry 
+weight: 20
+---
+
+The OpenTelemetry project aims to provide backwards compatibility with the
+[OpenTracing](https://opentracing.io) project, in order to ease migration to OpenTelemetry. 
+The intention is to allow OpenTracing instrumentation to be
+recorded using the OpenTelemetry SDK, and to allow both the OpenTelemetry 
+and the OpenTracing APIs to be used together in the same codebase.
+
+To accomplish this, each OpenTelemetry SDK provides and **OpenTracing Shim**, which can be installed
+in order to connect the SDK to the OpenTracing API. Please note that the OpenTracing Shim is not
+enabled by default.
+
+## Language version Support
+
+For reference see the following table, listing the OpenTracing
+and OpenTelemetry APIs minimum **language** version, as of
+January 2022:
+
+| Language   | OpenTracing API  | OpenTelemetry API |
+| ---------- | ---------------- | ----------------- |
+| Go         | 1.13             | 1.16              |
+| Java       | 7                | 8                 |
+| Python     | 2.7              | 3.6               |
+| Javascript | 6                | 8.5               |
+| DotNet     | 1.3              | 1.4               |
+| CPP        | 11               | 11                |
```

Comment:
Will this table might age quickly? If so, are you comfortable with that (since this isn't a blog page, but a main docs page).

Reply:
Yes, I expect this data to change rarely, if at all. Dating it is still helpful though.